py/desiutil/test/test_names.py

```diff
@@ -56,3 +56,8 @@ def test_radec_to_desiname_bad_values(self):
         with self.assertRaises(ValueError) as e:
             outnames = radec_to_desiname(ras, decs)
         self.assertEqual(str(e.exception), "NaN values detected in target_ra!")
```

Comment:
This and line 63 are fine as-is but if a well-meaning contributor changed the error message to be more informative they could break this test. It would be easy to address at that time, but we could also anticipate that now by only require specific substrings to be in `e`, or leave it as sufficient to just have it raise the correct error.

Reply:
I don't think this is a significant concern. If someone changes the code future, then they need to change the test. There's nothing unexpected about that.